.github/workflows/test_pr_and_main.yml

```diff
@@ -339,9 +349,9 @@ jobs:
         fi
         # HACK: Remove problem packages on conda+Linux
         if test "${{matrix.TARGET}}" == linux; then
-            EXCLUDE="casadi numdifftools pint $EXCLUDE"
+            EXCLUDE="casadi numdifftools $EXCLUDE"
         fi
-        EXCLUDE=`echo "$EXCLUDE" | xargs`
+        EXCLUDE=`echo "$EXCLUDE PyQt6 pyqt6" | xargs`
```

Comment:
For unconditional exclusions, can we specify them in a CONDA_EXCLUDE environment variable (like we do with PYPY_EXCLUDE)?